Traduku Ĉapitron Unuan (#1)
norwd/hpmor: chapters/hpmor-chapter-001.tex

Jen la salono de la domo, en kiu loĝas la eminenta Profesoro Mikael Veres\nobreakdash-Evanz, kaj sia edzino, S\nobreakdash-ino.~Petunja Evanz\nobreakdash-Veres, kaj siaj adoptita filo, Hari Ĝajmz Potr-Evanz-Veres.

Sur la tablo de la salono kuŝantas leteron, apud nestampita koverto el flaveta pergameno, adresita al \emph{S\nobreakdash-ro.~H.~Potr} per smeraldverda inko.

La Profesoro kaj sia edzino parolas akre unu al alia, sed ili ne laŭtas. La Profesoro opinias, ke kriado estas necivilizita.

“Vi ŝercas,” Mikael drias al Petunja. Lia tono indikis ke li tre timis ke ŝi estis serioza.

“Mia filino estis sorĉistino,” Petunja ripetis. Ŝi aspektis timigita, sed staris firme. “Ŝia edzo estis sorĉisto.”

> “This is absurd!” Mikael said sharply. “They were at our wedding—they visited for Christmas—”

Author: ```suggestion
“This is absurd!” Mikael said akre. “They were at our wedding—they visited for Christmas—”
```

“I told them you weren’t to know,” Petunja whispered. “But it’s true. I’ve seen things—”

The Professor rolled his eyes. “Dear, I understand that you’re not familiar with the sceptical literature. You may not realise how easy it is for a trained magician to fake the seemingly impossible. Remember how I taught Hari to bend spoons? If it seemed like they could always guess what you were thinking, that’s called cold reading—”

“It wasn’t bending spoons—”

“What was it, then?”

Petunja bit her lip. “I can’t just tell you. You’ll think I’m—” She swallowed. “Listen. Mikael. I wasn’t—always like this—” She gestured at herself, as though to indicate her lithe form. “Lili did this. Because I—because I \emph{begged} her. For years, I begged her. Lili had \emph{always} been prettier than me, and I’d…been mean to her, because of that, and then she got \emph{magic}, can you imagine how I felt? And I \emph{begged} her to use some of that magic on me so that I could be pretty too, even if I couldn’t have her magic, at least I could be pretty.”

Tears were gathering in Petunja’s eyes.

> “And Lili would tell me no, and make up the most ridiculous excuses, like the world would end if she were nice to her sister, or a centaur told her not to—the most ridiculous things, and I hated her for it. And when I had just graduated from university, I was going out with this boy, Vernon Dursley, he was fat and he was the only boy who would talk to me. And he said he wanted children, and that his first son would be named Dudley. And I thought to myself, \emph{what kind of parent names their child Dudley Dursley?} It was like I saw my whole future life stretching out in front of me, and I couldn’t stand it. And I wrote to my sister and told her that if she didn’t help me I’d rather just—”

Author: ```suggestion
“And Lili would tell me no, and make up the most ridiculous excuses, like the world would end if she were nice to her sister, or a centaur told her not to—the most ridiculous things, and I hated her for it. And when I had just graduated from university, I was going out with this boy, Vernon Dursley, he was fat and he was the only boy who would talk to me. And he said he wanted children, and that his first son would be named Dadli. And I thought to myself, \emph{what kind of parent names their child Dadli Dursli?} It was like I saw my whole future life stretching out in front of me, and I couldn’t stand it. And I wrote to my sister and told her that if she didn’t help me I’d rather just—”
```

“Darling,” Mikael said gently, “you got sick, you gained some weight while resting in bed, and your skin cleared up on its own. Or being sick made you change your diet—”

“She was a witch,” Petunja repeated. “I saw it.”

“Petunja,” Mikael said. The annoyance was creeping into his voice. “You \emph{know} that can’t be true. Do I really have to explain why?”

Petunja wrung her hands. She seemed to be on the verge of tears. “My love, I know I can’t win arguments with you, but please, you have to trust me on this—”

“\emph{Dad! Mum!}”

The two of them stopped and looked at Hari as though they’d forgotten there was a third person in the room.

Hari took a deep breath. “Mum, \emph{your} parents didn’t have magic, did they?”

“No,” Petunja said, looking puzzled.

“Then no-one in your family knew about magic when Lili got her letter. How did \emph{they} get convinced?”

> “Ah…” Petunja said. “They didn’t just send a letter. They sent a professor from Hogwarts. He—” Petunja’s eyes flicked to Mikael. “He showed us some magic.”

Author: ```suggestion
“Ah…” Petunja said. “They didn’t just send a letter. They sent a professor from Hogŭartso. He—” Petunja’s eyes flicked to Mikael. “He showed us some magic.”
```

> “Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogwarts professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”

Author: ```suggestion
“Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogŭartso professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”
```

Now his parents were getting into one of \emph{those} arguments again, one where his mother tried to make his father feel guilty, and his father tried to make his mother feel stupid.

“I’m going to go to my room,” Hari announced. His voice trembled a little. “Please try not to fight too much about this, Mum, Dad, we’ll know soon enough how it comes out, right?”

“Of course, Hari,” said his father, and his mother gave him a reassuring kiss, and then they went on fighting while Hari climbed the stairs to his bedroom.

He shut the door behind him and tried to think.

The funny thing was, he \emph{should} have agreed with Dad. No-one had ever seen any evidence of magic, and according to Mum, there was a whole magical world out there. How could anyone keep something like that a secret? More magic? That seemed like a rather suspicious sort of excuse.

It should have been a clean case for Mum joking, lying or being insane, in ascending order of awfulness. If Mum had sent the letter herself, that would explain how it arrived at the letterbox without a stamp. A little insanity was far, far less improbable than the universe really working like that.

> Except that some part of Hari was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogwarts School of Witchcraft and Wizardry.

Author: ```suggestion
Except that some part of Hari was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogŭartso School of Witchcraft and Wizardry.
```

Hari rubbed his forehead, grimacing. \emph{Don’t believe everything you think,} one of his books had said.

> But this bizarre certainty…Hari was finding himself just \emph{expecting} that, yes, a Hogwarts professor would show up and wave a wand and magic would come out. The strange certainty was making no effort to guard itself against falsification—wasn’t making excuses in advance for why there wouldn’t be a professor, or the professor would only be able to bend spoons.

Author: ```suggestion
But this bizarre certainty…Hari was finding himself just \emph{expecting} that, yes, a Hogŭartso professor would show up and wave a wand and magic would come out. The strange certainty was making no effort to guard itself against falsification—wasn’t making excuses in advance for why there wouldn’t be a professor, or the professor would only be able to bend spoons.
```

\emph{Where do you come from, strange little prediction?} Hari directed the thought at his brain. \emph{Why do I believe what I believe?}

Usually Hari was pretty good at answering that question, but in this particular case, he had no \emph{clue} what his brain was thinking.

Hari mentally shrugged. A flat metal plate on a door affords pushing, and a handle on a door affords pulling, and the thing to do with a testable hypothesis is to go and test it.

He took a piece of lined paper from his desk, and started writing.

\begin{writtenNote}

> \letterAddress{Dear Deputy Headmistress}

Author: ```suggestion
\letterAddress{Dear Deputy Lernejestrino}
```

\end{writtenNote}

Hari paused, reflecting; then discarded the paper for another, tapping another millimetre of graphite from his mechanical pencil. This called for careful calligraphy.

\begin{writtenNote}

> \letterAddress{Dear Deputy Headmistress Minerva Mikgonagol,}

Author: ```suggestion
\letterAddress{Dear Deputy Lernejestrino Minerva Mikgonagol,}
```

\letterAddress{Or Whomsoever It May Concern:}

> I recently received your letter of acceptance to Hogwarts, addressed to Mr~H.~Potr. You may not be aware that my genetic parents, Ĝajmz Potr and Lili Potr (formerly Lili Evanz) are dead. I was adopted by Lili’s sister, Petunja Evanz-Veres, and her husband, Mikael Veres-Evanz.

Author: ```suggestion
I recently received your letter of acceptance to Hogŭartso, addressed to Mr~H.~Potr. You may not be aware that my genetic parents, Ĝajmz Potr and Lili Potr (formerly Lili Evanz) are dead. I was adopted by Lili’s sister, Petunja Evanz-Veres, and her husband, Mikael Veres-Evanz.
```

> I am extremely interested in attending Hogwarts, conditional on such a place actually existing. Only my mother Petunja says she knows about magic, and she can’t use it herself. My father is highly sceptical. I myself am uncertain. I also don’t know where to obtain any of the books or equipment listed in your acceptance letter.

Author: ```suggestion
I am extremely interested in attending Hogŭartso, conditional on such a place actually existing. Only my mother Petunja says she knows about magic, and she can’t use it herself. My father is highly sceptical. I myself am uncertain. I also don’t know where to obtain any of the books or equipment listed in your acceptance letter.
```

> Mother mentioned that you sent a Hogwarts representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.

Author: ```suggestion
Mother mentioned that you sent a Hogŭartso representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.
```

\letterClosing[Sincerely,]{Hari Ĝajmz Potr-Evanz-Veres.}
\end{writtenNote}

> Hari added their current address, then folded up the letter and put it in an envelope, which he addressed to Hogwarts. Further consideration led him to obtain a candle and drip wax onto the flap of the envelope, into which, using a penknife’s tip, he impressed the initials H.J.P.E.V\@. If he was going to descend into this madness, he was going to do it with style.

Author: ```suggestion
Hari added their current address, then folded up the letter and put it in an envelope, which he addressed to Hogŭartso. Further consideration led him to obtain a candle and drip wax onto the flap of the envelope, into which, using a penknife’s tip, he impressed the initials H.J.P.E.V\@. If he was going to descend into this madness, he was going to do it with style.
```

Then he opened his door and went back downstairs. His father was sitting in the living-room and reading a book of higher maths to show how smart he was; and his mother was in the kitchen preparing one of his father’s favourite meals to show how loving she was. It didn’t look like they were talking to one another at all. As scary as arguments could be, \emph{not arguing} was somehow much worse.

> “Mum,” Hari said into the unnerving silence, “I’m going to test the hypothesis. According to your theory, how do I send an owl to Hogwarts?”

Author: ```suggestion
“Mum,” Hari said into the unnerving silence, “I’m going to test the hypothesis. According to your theory, how do I send an owl to Hogŭartso?”
```

His mother turned from the kitchen sink to stare at him, looking shocked. “I—I don’t know, I think you just have to own a magic owl.”

That should’ve sounded highly suspicious, \emph{oh, so there’s no way to test your theory then}, but the peculiar certainty in Hari seemed willing to stick its neck out even further.

> “Well, the letter got here somehow,” Hari said, “so I’ll just wave it around outside and call ‘letter for Hogwarts!’ and see if an owl picks it up. Dad, do you want to come and watch?”

Author: ```suggestion
“Well, the letter got here somehow,” Hari said, “so I’ll just wave it around outside and call ‘letter for Hogŭartso!’ and see if an owl picks it up. Dad, do you want to come and watch?”
```

His father shook his head minutely and kept on reading. \emph{Of course,} Hari thought to himself. Magic was a disgraceful thing that only stupid people believed in; if his father went so far as to \emph{test} the hypothesis, or even \emph{watch} it being tested, that would feel like \emph{associating} himself with that…

Only as Hari stumped out the back door, into the back garden, did it occur to him that if an owl \emph{did} come down and snatch the letter, he was going to have some trouble telling Dad about it.

\emph{But—well—that can’t \emph{really} happen, can it? No matter what my brain seems to believe. If an owl really comes down and grabs this envelope, I’m going to have worries a lot more important than what Dad thinks.}

Hari took a deep breath, and raised the envelope into the air.

He swallowed.

> Calling out \emph{Letter for Hogwarts!} while holding an envelope high in the air in the middle of your own back garden was…actually pretty embarrassing, now that he thought about it.

Author: ```suggestion
Calling out \emph{Letter for Hogŭartso!} while holding an envelope high in the air in the middle of your own back garden was…actually pretty embarrassing, now that he thought about it.
```

\emph{No. I’m better than Dad. I will use the scientific method even if it makes me feel stupid.}

“Letter—” Hari said, but it actually came out as more of a whispered croak.

> Hari steeled his will, and shouted into the empty sky, “\emph{Letter for Hogwarts! Can I get an owl?}”

Author: ```suggestion
Hari steeled his will, and shouted into the empty sky, “\emph{Letter for Hogŭartso! Can I get an owl?}”
```

“Hari?” asked a bemused woman’s voice, one of the neighbours.

Hari pulled down his hand like it was on fire and hid the envelope behind his back like it was drug money. His whole face was hot with shame.

An old woman’s face peered out from above the neighbouring fence, grizzled grey hair escaping from her hairnet. Mrs~Figg, the occasional babysitter. “What are you doing, Hari?”

“Nothing,” Hari said in a strangled voice. “Just—testing a really silly theory—”

> “Did you get your acceptance letter from Hogwarts?”

Author: ```suggestion
“Did you get your acceptance letter from Hogŭartso?”
```

Hari froze in place.

> “Yes,” Hari’s lips said a little while later. “I got a letter from Hogwarts. They say they want my owl by the 31st of July, but—”

Author: ```suggestion
“Yes,” Hari’s lips said a little while later. “I got a letter from Hogŭartso. They say they want my owl by the 31st of July, but—”
```

Jen la salono de la domo, en kiu loĝas la eminenta Profesoro Mikael Veres\nobreakdash-Evanz, kaj sia edzino, S\nobreakdash-ino.~Petunja Evanz\nobreakdash-Veres, kaj siaj adoptita filo, Hari Ĝajmz Potr-Evanz-Veres.

Sur la tablo de la salono kuŝantas leteron, apud nestampita koverto el flaveta pergameno, adresita al \emph{S\nobreakdash-ro.~H.~Potr} per smeraldverda inko.

La Profesoro kaj sia edzino parolas akre unu al alia, sed ili ne laŭtas. La Profesoro opinias, ke kriado estas necivilizita.

“Vi ŝercas,” Mikael drias al Petunja. Lia tono indikis ke li tre timis ke ŝi estis serioza.

“Mia filino estis sorĉistino,” Petunja ripetis. Ŝi aspektis timigita, sed staris firme. “Ŝia edzo estis sorĉisto.”

“Ĉi tio estas absurda!” Mikael diris akre. “Ili venis al nia geedziĝo—ili vizitis por Kristnasko—”

> “Mi diris al ilin ke vi ne devas scii,” Petunja flustris. “Sed estas la vero. Mi vidis aferojn—”

Author: ```suggestion
“Mi diris al ili, ke vi ne sciu,” flustris Petunja. “Sed ĝi estas vera. Mi vidis aferojn—”
```

The Professor turned and looked down at him, dismissive as usual. “Oh, come now, Hari. Really, \emph{magic}? I thought \emph{you’d} know better than to take this seriously, son, even if you’re only ten. Magic is just about the most unscientific thing there is!”

Hari’s mouth twisted bitterly. He was treated well, probably better than most genetic fathers treated their own children. Hari had been sent to the best primary schools—and when that didn’t work out, he was provided with tutors from the endless pool of starving students. Always Hari had been encouraged to study whatever caught his attention, bought all the books that caught his fancy, sponsored in whatever maths or science competitions he entered. He was given anything reasonable that he wanted, except, maybe, the slightest shred of respect. A Doctor teaching biochemistry at Oxford could hardly be expected to listen to the advice of a little boy. You would listen to Show Interest, of course; that’s what a Good Parent would do, and so, if you conceived of yourself as a Good Parent, you would do it. But take a ten-year-old \emph{seriously}? Hardly.

> Sometimes Hari wanted to scream at his father.

Author: ```suggestion
Kelkfoje Hari volis krii al sia patro.
```

Hari rubbed his forehead, grimacing. \emph{Don’t believe everything you think,} one of his books had said.

But this bizarre certainty…Hari was finding himself just \emph{expecting} that, yes, a Hogŭartso professor would show up and wave a wand and magic would come out. The strange certainty was making no effort to guard itself against falsification—wasn’t making excuses in advance for why there wouldn’t be a professor, or the professor would only be able to bend spoons.

\emph{Where do you come from, strange little prediction?} Hari directed the thought at his brain. \emph{Why do I believe what I believe?}

Usually Hari was pretty good at answering that question, but in this particular case, he had no \emph{clue} what his brain was thinking.

Hari mentally shrugged. A flat metal plate on a door affords pushing, and a handle on a door affords pulling, and the thing to do with a testable hypothesis is to go and test it.

He took a piece of lined paper from his desk, and started writing.

\begin{writtenNote}

> \letterAddress{Dear Deputy Lernejestrino}

Author: ```suggestion
\letterAddress{Estimata Deputita Lernejestrino}
```

\end{writtenNote}

Hari paused, reflecting; then discarded the paper for another, tapping another millimetre of graphite from his mechanical pencil. This called for careful calligraphy.

\begin{writtenNote}

> \letterAddress{Dear Deputy Lernejestrino Minerva Mikgonagol,}

Author: ```suggestion
\letterAddress{Estimata Deputita Lernejestrino Minerva Mikgonagol,}
```

> \letterAddress{Or Whomsoever It May Concern:}

Author: ```suggestion
\letterAddress{Aŭ Kiu Ajn Povas Esti Koncernita:}
```

I recently received your letter of acceptance to Hogŭartso, addressed to S\nobreakdash-ro.~H.~Potr. You may not be aware that my genetic parents, Ĝajmz Potr and Lili Potr (formerly Lili Evanz) are dead. I was adopted by Lili’s sister, Petunja Evanz-Veres, and her husband, Mikael Veres-Evanz.

I am extremely interested in attending Hogŭartso, conditional on such a place actually existing. Only my mother Petunja says she knows about magic, and she can’t use it herself. My father is highly sceptical. I myself am uncertain. I also don’t know where to obtain any of the books or equipment listed in your acceptance letter.

Mother mentioned that you sent a Hogŭartso representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.

\letterClosing[Sincerely,]{Hari Ĝajmz Potr-Evanz-Veres.}
\end{writtenNote}

> Hari added their current address, then folded up the letter and put it in an envelope, which he addressed to Hogŭartso. Further consideration led him to obtain a candle and drip wax onto the flap of the envelope, into which, using a penknife’s tip, he impressed the initials H.J.P.E.V\@. If he was going to descend into this madness, he was going to do it with style.

Author: ```suggestion
Hari added their current address, then folded up the letter and put it in an envelope, which he addressed to Hogŭartso. Further consideration led him to obtain a candle and drip wax onto the flap of the envelope, into which, using a penknife’s tip, he impressed the initials H.Ĝ.P.E.V\@. If he was going to descend into this madness, he was going to do it with style.
```

“Ĉi tio estas absurda!” Mikael diris akre. “Ili venis al nia geedziĝo—ili vizitis por Kristnasko—”

“Mi diris al ili, ke vi ne sciu,” flustris Petunja. “Sed ĝi estas vera. Mi vidis aferojn—”

The Professor rolled his eyes. “Dear, I understand that you’re not familiar with the sceptical literature. You may not realise how easy it is for a trained magician to fake the seemingly impossible. Remember how I taught Hari to bend spoons? If it seemed like they could always guess what you were thinking, that’s called cold reading—”

“It wasn’t bending spoons—”

“What was it, then?”

Petunja bit her lip. “I can’t just tell you. You’ll think I’m—” She swallowed. “Listen. Mikael. I wasn’t—always like this—” She gestured at herself, as though to indicate her lithe form. “Lili did this. Because I—because I \emph{begged} her. For years, I begged her. Lili had \emph{always} been prettier than me, and I’d…been mean to her, because of that, and then she got \emph{magic}, can you imagine how I felt? And I \emph{begged} her to use some of that magic on me so that I could be pretty too, even if I couldn’t have her magic, at least I could be pretty.”

Tears were gathering in Petunja’s eyes.

> “And Lili would tell me no, and make up the most ridiculous excuses, like the world would end if she were nice to her sister, or a centaur told her not to—the most ridiculous things, and I hated her for it. And when I had just graduated from university, I was going out with this boy, Vernon Dursley, he was fat and he was the only boy who would talk to me. And he said he wanted children, and that his first son would be named Dadli. And I thought to myself, \emph{what kind of parent names their child Dadli Dursli?} It was like I saw my whole future life stretching out in front of me, and I couldn’t stand it. And I wrote to my sister and told her that if she didn’t help me I’d rather just—”

Author: ```suggestion
“And Lili would tell me no, and make up the most ridiculous excuses, like the world would end if she were nice to her sister, or a centaur told her not to—the most ridiculous things, and I hated her for it. And when I had just graduated from university, I was going out with this boy, Vernon' Dursli, he was fat and he was the only boy who would talk to me. And he said he wanted children, and that his first son would be named Dadli. And I thought to myself, \emph{what kind of parent names their child Dadli Dursli?} It was like I saw my whole future life stretching out in front of me, and I couldn’t stand it. And I wrote to my sister and told her that if she didn’t help me I’d rather just—”
```

Lia nomo estas "Vernon(o)" ne "Verno".

Petunja stopped.

“Anyway,” Petunja said, her voice small, “she gave in. She told me it was dangerous, and I said I didn’t care any more, and I drank this potion and I was sick for weeks, but when I got better my skin cleared up and I finally filled out and…I was beautiful, people were \emph{nice} to me,” her voice broke, “and after that I couldn’t hate my sister any more, especially when I learned what her magic brought her in the end—”

“Darling,” Mikael said gently, “you got sick, you gained some weight while resting in bed, and your skin cleared up on its own. Or being sick made you change your diet—”

> “She was a witch,” Petunja repeated. “I saw it.”

Author: ```suggestion
“Ŝi estis sorĉistino,” Petunja ripetis. “Mi vidis tion.”
```

...aŭ eble...?

```suggestion
“Ŝi estis sorĉistino,” Petunja ripetis. “Mi vidis ĝin.”
```

“Petunja,” Mikael said. The annoyance was creeping into his voice. “You \emph{know} that can’t be true. Do I really have to explain why?”

Petunja wrung her hands. She seemed to be on the verge of tears. “My love, I know I can’t win arguments with you, but please, you have to trust me on this—”

> “\emph{Dad! Mum!}”

Author: ```suggestion
“\emph{Paĉjo! Panjo!}”
```

The two of them stopped and looked at Hari as though they’d forgotten there was a third person in the room.

Hari took a deep breath. “Mum, \emph{your} parents didn’t have magic, did they?”

“No,” Petunja said, looking puzzled.

“Then no-one in your family knew about magic when Lili got her letter. How did \emph{they} get convinced?”

“Ah…” Petunja said. “They didn’t just send a letter. They sent a professor from Hogŭartso. He—” Petunja’s eyes flicked to Mikael. “He showed us some magic.”

> “Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogŭartso professor here and see the magic for ourselves, and Dad will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”

Author: ```suggestion
“Then you don’t have to fight over this,” Hari said firmly. Hoping against hope that this time, just this once, they would listen to him. “If it’s true, we can just get a Hogŭartso professor here and see the magic for ourselves, and Paĉjo will admit that it’s true. And if not, then Mum will admit that it’s false. That’s what the experimental method is for, so that we don’t have to resolve things just by arguing.”
```

The Professor turned and looked down at him, dismissive as usual. “Oh, come now, Hari. Really, \emph{magic}? I thought \emph{you’d} know better than to take this seriously, son, even if you’re only ten. Magic is just about the most unscientific thing there is!”

Hari’s mouth twisted bitterly. He was treated well, probably better than most genetic fathers treated their own children. Hari had been sent to the best primary schools—and when that didn’t work out, he was provided with tutors from the endless pool of starving students. Always Hari had been encouraged to study whatever caught his attention, bought all the books that caught his fancy, sponsored in whatever maths or science competitions he entered. He was given anything reasonable that he wanted, except, maybe, the slightest shred of respect. A Doctor teaching biochemistry at Oxford could hardly be expected to listen to the advice of a little boy. You would listen to Show Interest, of course; that’s what a Good Parent would do, and so, if you conceived of yourself as a Good Parent, you would do it. But take a ten-year-old \emph{seriously}? Hardly.

Kelkfoje Hari volis krii al sia patro.

> “Mum,” Hari said. “If you want to win this argument with Dad, look in chapter two of the first book of the Feynman Lectures on Physics. There’s a quote there about how philosophers say a great deal about what science absolutely requires, and it is all wrong, because the only rule in science is that the final arbiter is observation—that you just have to look at the world and report what you see. Um…off the top of my head I can’t think of where to find something about how it’s an ideal of science to settle things by experiment instead of arguments—”

Author: ```suggestion
“Mum,” Hari said. “If you want to win this argument with Paĉjo, look in chapter two of the first book of the Feynman Lectures on Physics. There’s a quote there about how philosophers say a great deal about what science absolutely requires, and it is all wrong, because the only rule in science is that the final arbiter is observation—that you just have to look at the world and report what you see. Um…off the top of my head I can’t think of where to find something about how it’s an ideal of science to settle things by experiment instead of arguments—”
```

His mother looked down at him and smiled. “Thank you, Hari. But—” her head rose back up to stare at her husband. “I don’t want to win an argument with your father. I want my husband to, to listen to his wife who loves him, and trust her just this once—”

Hari closed his eyes briefly. \emph{Hopeless.} Both of his parents were just hopeless.

Now his parents were getting into one of \emph{those} arguments again, one where his mother tried to make his father feel guilty, and his father tried to make his mother feel stupid.

> “I’m going to go to my room,” Hari announced. His voice trembled a little. “Please try not to fight too much about this, Mum, Dad, we’ll know soon enough how it comes out, right?”

Author: ```suggestion
“I’m going to go to my room,” Hari announced. His voice trembled a little. “Please try not to fight too much about this, Mum, Paĉjo, we’ll know soon enough how it comes out, right?”
```

“Of course, Hari,” said his father, and his mother gave him a reassuring kiss, and then they went on fighting while Hari climbed the stairs to his bedroom.

He shut the door behind him and tried to think.

> The funny thing was, he \emph{should} have agreed with Dad. No-one had ever seen any evidence of magic, and according to Mum, there was a whole magical world out there. How could anyone keep something like that a secret? More magic? That seemed like a rather suspicious sort of excuse.

Author: ```suggestion
The funny thing was, he \emph{should} have agreed with Paĉjo. No-one had ever seen any evidence of magic, and according to Mum, there was a whole magical world out there. How could anyone keep something like that a secret? More magic? That seemed like a rather suspicious sort of excuse.
```

> It should have been a clean case for Mum joking, lying or being insane, in ascending order of awfulness. If Mum had sent the letter herself, that would explain how it arrived at the letterbox without a stamp. A little insanity was far, far less improbable than the universe really working like that.

Author: ```suggestion
It should have been a clean case for Panjo joking, lying or being insane, in ascending order of awfulness. If Panjo had sent the letter herself, that would explain how it arrived at the letterbox without a stamp. A little insanity was far, far less improbable than the universe really working like that.
```

Except that some part of Hari was utterly convinced that magic was real, and had been since the instant he saw the putative letter from the Hogŭartso School of Witchcraft and Wizardry.

Hari rubbed his forehead, grimacing. \emph{Don’t believe everything you think,} one of his books had said.

But this bizarre certainty…Hari was finding himself just \emph{expecting} that, yes, a Hogŭartso profesoro would show up and wave a wand and magic would come out. The strange certainty was making no effort to guard itself against falsification—wasn’t making excuses in advance for why there wouldn’t be a profesoro, or the profesoro would only be able to bend spoons.

> \emph{Where do you come from, strange little prediction?} Hari directed the thought at his brain. \emph{Why do I believe what I believe?}

Author: ```suggestion
\emph{Where do you come from, strange little prediction?} Hari directed the thought at his brain. \emph{Kial mi kredas tion, kion mi kredas?}
```

Mother mentioned that you sent a Hogŭartso representative to Lili Potr (then Lili Evanz) in order to demonstrate to her family that magic was real, and, I presume, help Lili obtain her school materials. If you could do this for my own family it would be extremely helpful.

\letterClosing[Sincerely,]{Hari Ĝajmz Potr-Evanz-Veres.}
\end{writtenNote}

Hari added their current address, then folded up the letter and put it in an envelope, which he addressed to Hogŭartso. Further consideration led him to obtain a candle and drip wax onto the flap of the envelope, into which, using a penknife’s tip, he impressed the initials H.Ĝ.P.E.V\@. If he was going to descend into this madness, he was going to do it with style.

Then he opened his door and went back downstairs. His father was sitting in the living-room and reading a book of higher maths to show how smart he was; and his mother was in the kitchen preparing one of his father’s favourite meals to show how loving she was. It didn’t look like they were talking to one another at all. As scary as arguments could be, \emph{not arguing} was somehow much worse.

> “Mum,” Hari said into the unnerving silence, “I’m going to test the hypothesis. According to your theory, how do I send an owl to Hogŭartso?”

Author: ```suggestion
“Panjo,” Hari said into the unnerving silence, “I’m going to test the hypothesis. According to your theory, how do I send an owl to Hogŭartso?”
```

His mother turned from the kitchen sink to stare at him, looking shocked. “I—I don’t know, I think you just have to own a magic owl.”

That should’ve sounded highly suspicious, \emph{oh, so there’s no way to test your theory then}, but the peculiar certainty in Hari seemed willing to stick its neck out even further.

> “Well, the letter got here somehow,” Hari said, “so I’ll just wave it around outside and call ‘letter for Hogŭartso!’ and see if an owl picks it up. Dad, do you want to come and watch?”

Author: ```suggestion
“Well, the letter got here somehow,” Hari said, “so I’ll just wave it around outside and call ‘letter for Hogŭartso!’ and see if an owl picks it up. Paĉjo, do you want to come and watch?”
```

His father shook his head minutely and kept on reading. \emph{Of course,} Hari thought to himself. Magic was a disgraceful thing that only stupid people believed in; if his father went so far as to \emph{test} the hypothesis, or even \emph{watch} it being tested, that would feel like \emph{associating} himself with that…

> Only as Hari stumped out the back door, into the back garden, did it occur to him that if an owl \emph{did} come down and snatch the letter, he was going to have some trouble telling Dad about it.

Author: ```suggestion
Only as Hari stumped out the back door, into the back garden, did it occur to him that if an owl \emph{did} come down and snatch the letter, he was going to have some trouble telling Paĉjo about it.
```

> \emph{But—well—that can’t \emph{really} happen, can it? No matter what my brain seems to believe. If an owl really comes down and grabs this envelope, I’m going to have worries a lot more important than what Dad thinks.}

Author: ```suggestion
\emph{But—well—that can’t \emph{really} happen, can it? No matter what my brain seems to believe. If an owl really comes down and grabs this envelope, I’m going to have worries a lot more important than what Paĉjo thinks.}
```

Hari took a deep breath, and raised the envelope into the air.

He swallowed.

Calling out \emph{Letter for Hogŭartso!} while holding an envelope high in the air in the middle of your own back garden was…actually pretty embarrassing, now that he thought about it.

> \emph{No. I’m better than Dad. I will use the scientific method even if it makes me feel stupid.}

Author: ```suggestion
\emph{No. I’m better than Paĉjo. I will use the scientific method even if it makes me feel stupid.}
```

\chapter{Tago de Tre Malgranda Probableco}

\begin{chapterOpeningQuote}
\noindent
Sub la lunlumo briletas eta arĝenta fragmento, frakcio de linio…

\vspace*{2ex}
(nigraj roboj, falantaj)

\vspace*{2ex}

> …sango verŝiĝas je litroj, kaj iu kriegas vorton.

Author: ```suggestion
…sango elverŝiĝas po litroj, kaj iu kriegas vorton.
```

\end{chapterOpeningQuote}

\lettrine{Ĉ}{iu} colo de muro-spaco estas kovrita per libro-ŝrankaĵo.
Ĉiu libroŝranko havas ses bretojn, ĉiu atinganta preskaŭ ĝis la plafono.

> Kelkaj librobretoj estas stakataj ĝis la rando de durkovraj libroj: scienco, matematiko, historio, kaj ĉio alia.

Author: ```suggestion
Kelkaj librobretoj estas stakataj ĝis la rando per durkovraj libroj: scienco, matematiko, historio, kaj ĉio alia.
```

Aliaj bretoj havas du tavolojn de paperkovraj sciencfikciaĵoj, kun la malantaŭa tavolo de libroj apogitaj sur malnovaj naztukskatoloj aŭ lignotabuloj, por vidi la malantaŭan tavolon de libroj super la libroj antaŭaj.
Kaj ankoraŭ ne sufiĉas.
Libroj estas superfluantaj sur la tablojn kaj la sofojn kaj faras amasetojn sub la fenestroj.

> Jen la salono de la domo, en kiu loĝas la eminenta Profesoro Mikael Veres\nobreakdash-Evanz, kaj sia edzino, S\nobreakdash-ino.~Petunja Evanz\nobreakdash-Veres, kaj siaj adoptita filo, Hari Ĝajmz Potr\nobreakdash-Evanz\nobreakdash-Veres.

Author: ```suggestion
Jen la salono de la domo, en kiu loĝas la eminenta Profesoro Mikael Veres\nobreakdash-Evanz, kaj sia edzino, S\nobreakdash-ino.~Petunja Evanz\nobreakdash-Veres, kaj sia adoptita filo, Hari Ĝajmz Potr\nobreakdash-Evanz\nobreakdash-Veres.
```

Sur la tablo de la salono estas kuŝanta leteron, apud nestampita koverto el flaveta pergameno, adresita al \emph{S\nobreakdash-ro.~H.~Potr} per smeralda-verda inko.

La Profesoro kaj sia edzino parolas akre unu al alia, sed ili ne laŭtas.

> La Profesoro opinias, ke kriado estas necivilizita.

Author: ```suggestion
La Profesoro opinias, ke kriadi estas necivilizita.
```

> “Vi ŝercas,” Mikael diras al Petunja.

Author: ```suggestion
“Vi estas ŝercanta,” Mikael diras al Petunja.
```

> “Vi estas ŝercanta,” Mikael diras al Petunja.

Author: ```suggestion
“Vi estas ŝercanta,” Mikael diris al Petunja.
```

> Lia tono indikis ke li tre timis ke ŝi estis serioza.

Author: ```suggestion
Lia tono indikis, ke li tre timis, ke ŝi estas serioza.
```

> “Mia filino estis sorĉistino,” Petunja ripetis.

Author: ```suggestion
“Mia fratino estis sorĉistino,” Petunja ripetis.
```

Ŝi aspektis timigita, sed staris firme.
“Ŝia edzo estis sorĉisto.”

“Ĉi tio estas absurda!” Mikael diris akre.
“Ili venis al nia geedziĝo—ili vizitis por Kristnasko—”

> “Mi diris al ili, ke vi ne sciu,” flustris Petunja.

Author: ```suggestion
“Mi diris al ili, ke vi devas ne scii,” flustris Petunja.
```

“Sed estas la vero. Mi vidis aferojn—”

La profesoro rulis la okulojn.
“Kara, mi komprenas, ke vi ne konas la skeptikan literaturon.
Vi eble ne rimarkas, kiel facile estas por trejnita iluziisto falsi la ŝajne neeblan.
Ĉu vi memoras, kiel mi instruis Hari fleksi kulerojn?
Se ŝajnis, ke ili ĉiam povis diveni, kion vi pensis, tio nomiĝas malvarma legado—”

“Ne estis fleksado de kuleroj—”

“Kio estis do?”

Petunja mordis la lipon.
“Mi simple ne povas diri al vi.
Vi kredos, ke mi estas—”
Ŝi glutis.
“Aŭskultu.
Mikael.
Mi ne estis—ĉiam tia—”
Ŝi gestis al si, kvazaŭ por indiki sian sveltan formon.
“Lili faris ĉi tio.
Ĉar mi—ĉar mi \emph{petegis} ŝin.
Dum jaroj, mi petegis ŝin.
Lili \emph{ĉiam} estis pli bela ol mi, kaj mi estis…nebonkora al ŝi, pro tio, kaj tiam ŝi akiris \emph{magion}, ĉu vi povas imagi kiel mi sentis min?
Kaj mi \emph{petegis} ŝin uzi iom da tiu magio sur mi, por ke mi estu bela ankaŭ, eĉ se mi ne povus havi ŝian magion, almenaŭ mi povus esti bela.”

Larmoj kolektiĝis en la okuloj de Petunja.

> “Kaj Lili diris al mi ne, kaj elpensis la plej ridindajn ekskuzojn, kiel la mondo finiĝus se ŝi estus afabla al sia fratino, aŭ centaŭro diris al ŝi ne—la plej ridindajn aferojn, kaj mi malamis ŝin pro tio.

Author: ```suggestion
“Kaj Lili diris al mi ne, kaj elpensis la plej ridindajn ekskuzojn, kvazaŭ la mondo finiĝus, se ŝi estus afabla al sia fratino, aŭ centaŭro diris al ŝi ne—la plej ridindajn aferojn, kaj mi malamis ŝin pro tio.
```

> Kaj kiam mi ĵus diplomiĝis el universitato, mi geamikis kun knabo, Vernon’ Durslij, li estis dika kaj li estis la nura knabo kiu dirus al mi.

Author: ```suggestion
Kaj kiam mi ĵus diplomiĝis el universitato, mi geamantis kun knabo, Vernon’ Durslij, li estis dika kaj li estis la nura knabo kiu parolus al mi.
```

Author: Mi havas ankoraŭ demandojn pri la uzado de 'geamiki' kiel traduko de 'was going out with' / 'dating'.

> Kaj kiam mi ĵus diplomiĝis el universitato, mi geamantis kun knabo, Vernon’ Durslij, li estis dika kaj li estis la nura knabo kiu parolus al mi.

Author: ```suggestion
Kaj kiam mi ĵus diplomiĝis el universitato, mi havis koramikon, Vernon’ Durslij, li estis dika kaj li estis la nura knabo kiu parolus al mi.
```

> Li volis infanojn, kaj ke lia unua filo nomiĝus Dadli.

Author: ```suggestion
Li volis infanojn, kaj ke lia unua filo nomiĝus Dadlij.
```

> Kaj mi pensis al mi, \emph{kia gepatro nomiĝas sia infano Dadlij Durslij?}

Author: ```suggestion
Kaj mi pensis al mi, \emph{kia gepatro nomas sian infanon Dadlij Durslij?}
```

> Estis kvazaŭ mi vidanta mian tutan estontan vivon streĉantan antaŭ mi, kaj mi ne povis elteni ĝin.

Author: ```suggestion
Estis kvazaŭ mi estis vidanta mian tutan estontan vivon streĉantan antaŭ mi, kaj mi ne povis elteni ĝin.
```

...aux...

```suggestion
Estis kvazaŭ mi vidantis mian tutan estontan vivon streĉantan antaŭ mi, kaj mi ne povis elteni ĝin.
```

...aux...

```suggestion
Kvazaŭ mi estis vidanta mian tutan estontan vivon streĉantan antaŭ mi, kaj mi ne povis elteni ĝin.
```

Author: Ne, vi ĉiuj malpravas!

```suggestion
Estis kvazaŭ mi povus vidi mian tutan estontan vivon streĉantan antaŭ mi, kaj mi ne povis elteni ĝin.
```

Kaj mi skribis al mia fratino kaj diris al ŝi, ke se ŝi ne helpus min, mi preferus simple—”

Petunja ĉesis.

“Ĉiuokaze,” Petunja diris kun malgranda voĉo, “ŝi cedis.

> Ŝi diris al mi ke ĝi estis danĝera, kaj mi diris ke mi ne plu zorgis, kaj mi trinkis pocion kaj mi malsanis dum semajnoj, sed kiam mi resaniĝis mia haŭto klariĝis kaj mi finfine plenigis kaj…mi estis bela, oni estis \emph{agrable} al mi,” ŝia voĉo rompis, “kaj poste mi ne plu povis malami mian filinon, ĉefe kiam mi lernis kion ŝian magion alportis ŝin fine—”

Author: ```suggestion
Ŝi diris al mi ke ĝi estis danĝera, kaj mi diris ke mi ne plu zorgis, kaj mi trinkis pocion kaj mi malsanis dum semajnoj, sed kiam mi resaniĝis mia haŭto klariĝis kaj mi finfine plenigis kaj…mi estis bela, oni estis \emph{agrable} al mi,” ŝia voĉo rompis, “kaj poste mi ne plu povis malami mian fratinon, ĉefe kiam mi lernis kion ŝian magion alportis ŝin fine—”
```

> Ŝi diris al mi ke ĝi estis danĝera, kaj mi diris ke mi ne plu zorgis, kaj mi trinkis pocion kaj mi malsanis dum semajnoj, sed kiam mi resaniĝis mia haŭto klariĝis kaj mi finfine plenigis kaj…mi estis bela, oni estis \emph{agrable} al mi,” ŝia voĉo rompis, “kaj poste mi ne plu povis malami mian fratinon, ĉefe kiam mi lernis kion ŝian magion alportis ŝin fine—”

Author: ```suggestion
Ŝi diris al mi ke ĝi estis danĝera, kaj mi diris ke mi ne plu zorgis, kaj mi trinkis pocion kaj mi malsanis dum semajnoj, sed kiam mi resaniĝis, mia haŭto fariĝis klara kaj mi finfine plenigis kaj…mi estis bela, oni estis \emph{agrable} al mi,” ŝia voĉo rompis, “kaj poste mi ne plu povis malami mian fratinon, ĉefe kiam mi lernis kion ŝian magion alportis ŝin fine—”
```

> “Karulino,” Mikael diris milde, “vi malsaniĝis, vi akiris iom da pezo dum ripozinta en lito, kaj via haŭto klariĝis memstare.

Author: ```suggestion
“Karulino,” Mikael diris milde, “vi malsaniĝis, vi akiris iom da pezo dum ripozinta en lito, kaj via hhaŭto fariĝis klara memstare.
```

> “Karulino,” Mikael diris milde, “vi malsaniĝis, vi akiris iom da pezo dum ripozinta en lito, kaj via hhaŭto fariĝis klara memstare.

Author: ```suggestion
“Karulino,” Mikael diris milde, “vi malsaniĝis, vi akiris iom da pezo dum ripozinta en lito, kaj via haŭto fariĝis klara memstare.
```

Aŭ esti malsana igis vin ŝanĝi vian dieton—”

“Ŝi estis sorĉistino,” Petunja ripetis.
“Mi vidis tion.”

“Petunja,” Mikael diris.
La ĝeno estis rampanta en sian voĉon.
“Vi \emph{scias} ke tio ne povas esti vera.
Certe mi ne bezonas klarigi kial?”

Petunja tordis siajn manojn.
Ŝi ŝajnis esti ĉe larmoj.
“Mia amo, mi scias ke mi ne povas venki argumentojn kontraŭ vi, sed bonvolu, vi devas fidi min pri ĉi tio—”

“\emph{Paĉjo! Panjo!}”

Ili du ĉesis kaj rigardis Hari kvazaŭ ili forgesis ke estis tria homo en la ĉambro.

Hari profunde enspiris.
“Panjo, \emph{viaj} gepatroj ne havis magion, ĉu ne?”

“Ne,” Petunja diris, aspektanta perplekse.

> “Tial neniu en via familio sciis pri magio kiam Lili ricevis sian leteron.

Author: ```suggestion
“Tial, neniu en via familio sciis pri magio kiam Lili ricevis sian leteron.
```

Kiel \emph{ili} konvinkiĝis?”

“Ha…” Petunja diris.
“Ili ne nur sendis leteron.
Ili sendis profesoron de Hogŭartso.
Li—” la okuloj de Petunja ekflugis al Mikael.
“Li montris al ni iom da magio.”

> “Tial vi ne devas batali pri ĉi tio,” Hari diris firme.

Author: ```suggestion
“Tial, vi ne devas batali pri ĉi tio,” Hari diris firme.
```

Esperante kontraŭ espero ke ĉi-foje, nur ĉi-foje, ili aŭskultus al li.
“Se ĝi estas vera, ni povas simple alporti Hogŭartsan profesoron ĉi tien kaj vidi la magion por ni mem, kaj Paĉjo akceptos ke ĝi estas vera.
Kaj se ne, tiam Panjo akceptos ke ĝi estas malvera.

> Tio estas por kio la eksperimenta metodo ekzistas, do ke ni ne devu solvi aferojn nur per argumentado.”

Author: ```suggestion
Tio estas por kio la eksperimenta metodo ekzistas, do ke ni ne bezonu solvi aferojn nur per argumentado.”
```

La profesoro turnis sin kaj rigardis malsupren al li, arogante kiel kutima.
“Ho, estu racia, Hari.
Ĉu vere, \emph{magio}?
Mi pensis ke \emph{vi} scius pli bone ol preni ĉi tion serioze, filo, eĉ se vi estas nur dekjara.

> Magio estas proksimume la plej nescienca afero kio estas!”

Author: ```suggestion
Magio estas proksimume la plej nescienca afero sur la tero!”
```

La buŝo de Hari maldolĉe tordis.
Li estis traktita bone, eble pli bone ol plejparto de genetikaj patroj traktis siajn proprajn infanojn.

> Hari estis sendita al la plej bonajn bazlernejojn—kaj kiam tio ne funkciis, al li estis provizitaj gvidinstruistoj de la senfina provizo de malsataj studentoj.

Author: ```suggestion
Hari estis sendita al la plej bonajn bazlernejojn—kaj kiam tio ne funkciis, al li estis provizitaj gvidinstruistoj el la senfina provizo de malsataj studentoj.
```

> Ĉiam Hari estis kuraĝigita studi kion ajn kaptis lian atenton, por li estis aĉetitaj ĉiuj libroj kiuj kaptis lian fantazion, li estis sponsorinta en kiaj ajn matematikaj aŭ sciencaj konkursoj li eniris.

Author: ```suggestion
Ĉiam Hari estis kuraĝigita studi kion ajn kaptis lian atenton, ĉiuj libroj kiuj kaptis lian fantazion estis aĉetitaj por li, li estis sponsorinta en kiaj ajn matematikaj aŭ sciencaj konkursoj li eniris.
```

> Ĉiam Hari estis kuraĝigita studi kion ajn kaptis lian atenton, ĉiuj libroj kiuj kaptis lian fantazion estis aĉetitaj por li, li estis sponsorinta en kiaj ajn matematikaj aŭ sciencaj konkursoj li eniris.

Author: ```suggestion
Ĉiam Hari estis kuraĝigita studi kion ajn kaptis lian atenton, ĉiuj libroj kiuj kaptis lian fantazion estis aĉetitaj por li, li estis sponsorita en kiaj ajn matematikaj aŭ sciencaj konkursoj li eniris.
```

Li ricevis ion ajn akcepteblan, kion li volis, krom, eble, la plej malgrandan peceton de respekto.
Doktoro instruante biokemion ĉe Oksfordo apenaŭ povus esti atendita por aŭskulti la konsilon de eta knabo.
Vi aŭskultus por ke Montru Intereson, certe; tio estas kio Bona Gepatro farus, kaj do, se vi konceptis vin mem tiel Bona Gepatro, vi farus tiel.
Sed ĉu vi prenas dek-jarulon \emph{serioze}?
Apenaŭ.

Kelkfoje Hari volis krii al sia patro.

“Panjo,” Hari diris.

> “Se vi volas venki ĉi tiun argumenton kontraŭ Paĉjo, rigardu la duan ĉapitron de la unua libro de \emph{La Prelegoj de Fajnman pri Fiziko}.

Author: ```suggestion
“Se vi volas gajni ĉi tiun argumenton kontraŭ Paĉjo, rigardu la duan ĉapitron de la unua libro de \emph{La Prelegoj de Fajnman pri Fiziko}.
```

Estas citaĵo tie pri kiel filozofoj diras multon pri tio, kion scienco absolute bezonas, kaj tio estas tute erara, ĉar la sola regulo en scienco estas ke la fina arbitro estas observado—ke vi simple devu rigardi la mondon kaj raportu tion, kion vi vidas.
Nu…de la supro de mia kapo, mi ne povas pensi kie oni povas trovi ion pri kiel estas idealo de scienco solvi aferojn per eksperimentoj anstataŭ per argumentoj—”

Lia patrino rigardis malsupren al li kaj ridetis.

> “Dankon, Hari.
> Sed—” ŝia kapo leviĝis por fiksrigardi sian edzon.
> “Mi ne volas venki argumenton kontraŭ via patro.
> Mi volas ke, ke mia edzo aŭskultu sian edzinon kiu amas lin, kaj fidi ŝin nur ĉi-foje—”

Author: ```suggestion
“Dankon, Hari.
Sed—” ŝia kapo leviĝis por fiksrigardi sian edzon.
“Mi ne volas gajni argumenton kontraŭ via patro.
Mi volas ke, ke mia edzo aŭskultu sian edzinon kiu amas lin, kaj fidi ŝin nur ĉi-foje—”
```

Hari mallonge fermis la okulojn.

> \emph{Senespera.}

Author: ```suggestion
\emph{Senesperaj.}
```

> Ambaŭ de liaj gepatroj estis simple senespera.

Author: ```suggestion
Ambaŭ de liaj gepatroj estis simple senesperaj.
```

> Nun liaj gepatroj estis enirantaj en unu el \emph{tiaj} argumentoj denove, patrino provis fari ke patro sentu sin kulpa, kaj patro provis fari ke patrino sentu sin stulta.

Author: Ĉu eble "enirantaj" ne estas la plej bona traduko de "getting in to"? Mi pensas ke tiu estas tro laŭvorte anstataŭ laŭsence.

Author: Vidu fe51953

> “Mi iru al mia ĉambro,” Hari anoncis.
> Lia voĉo tremis iom.
> “Mi petas, provu ne batali tro pri ĉi tio, Panjo, Paĉjo, ni scios sufiĉe baldaŭ kiel ĝi rezultos, ĉu ne?”

Author: ```suggestion
“Mi iras al mia ĉambro,” Hari anoncis.
Lia voĉo tremis iom.
“Mi petas, provu ne batali tro pri ĉi tio, Panjo, Paĉjo, ni scios sufiĉe baldaŭ kiel ĝi rezultos, ĉu ne?”
```

> “Kompreneble, Hari,” diris lia patro, kaj lia patrino donis al li trankviligan kison, kaj poste, ili daŭrigis batali, dum li supreniris la ŝtuparon al sia dormoĉambro.

Author: ```suggestion
“Jes certe, Hari,” diris lia patro, kaj lia patrino donis al li trankviligan kison, kaj poste, ili daŭrigis batali, dum li supreniris la ŝtuparon al sia dormoĉambro.
```

Li fermis la pordon malantaŭ lin kaj provis pensi.

> La amuza afero estis, ke li \emph{devus} konsentis kun Paĉjo.
> Neniu iam vidis ajnan pruvon de magio, kaj laŭ Panjo, estis tuta magia mondo.
> Kiel oni povus konservi ion tian sekrete?
> Pli da magio, ĉu?
> Tio ŝajnis kiel sufiĉe suspektinda speco de ekskuzo.

Author: ```suggestion
La amuza afero estis, ke li \emph{devus} konsenti kun Paĉjo.
Neniu iam vidis ajnan pruvon de magio, kaj laŭ Panjo, estis tuta magia mondo.
Kiel oni povus konservi ion tian sekretan?
Pli da magio, ĉu?
Tio ŝajnis kiel sufiĉe suspektinda speco de ekskuzo.
```

> Ĝi devus esti pura kazo de Panjo ŝercas, mensogas, aŭ frenezas, en kreskanta ordo de terureco.

Author: ```suggestion
Ĝi devus esti pura kazo ke Panjo estas ŝercanta, mensoganta, aŭ frenezanta, en kreskanta ordo de terureco.
```

Se Panjo sendus la leteron mem, tio eksplikus kiel ĝi alvenis ĉe la leterkesto sen poŝtmarko.
Iom da frenezo estis multe, multe pli verŝajna ol ke la universo vere funkciis tiel.

Escepte, iu parto de Hari estis tute konvinkita ke magio estis reala, kaj estis tiel ekde la momenteto li vidis la supozitan leteron de la Hogŭartsa Lernejo de Sorĉad’ kaj Sorĉism’.

Hari frotis la frunton, grimacante.
\emph{Ne kredu ĉion, kion vi pensas,} unu el liaj libroj diris.

> Sed ĉi tiu bizara certeco…Hari trovis sin simple \emph{atendanta} ke, jes, profesoro de Hogŭartso aperos kaj svingos vergon kaj magio eliros.

Author: ```suggestion
Sed ĉi tiu bizara certeco…li trovis sin simple \emph{atendanta} ke, jes, profesoro de Hogŭartso aperos kaj svingos vergon kaj magio eliros.
```

> Sed ĉi tiu bizara certeco…Hari trovis sin nur \emph{atendante} ke, jes, profesoro de Hogŭartso aperos kaj svingos vergon kaj magio eliros.
> La stranga certeco tute ne provis gardi sin kontraŭ estante pruvita malvera—ĝi ne faris ekskuzoj anticipe por kial ne estus profesoro, aŭ la profesoro nur povus fleksi kulerojn.

Author: ```suggestion
Sed ĉi tiu bizara certeco…Hari trovis sin simple \emph{atendanta} ke, jes, profesoro de Hogŭartso aperos kaj svingos vergon kaj magio eliros.
La stranga certeco tute ne provis gardi sin kontraŭ estante pruvita malvera—ĝi ne faris ekskuzoj anticipe por kial ne estus profesoro, aŭ la profesoro nur povus fleksi kulerojn.
```

> La stranga certeco tute ne provis gardi sin kontraŭ estante pruvita malvera—ĝi ne faris ekskuzoj anticipe por kial ne estus profesoro, aŭ la profesoro nur povus fleksi kulerojn.

Author: ```suggestion
La stranga certeco tute ne provis gardi sin kontraŭ estante pruvita malvera—ĝi ne faris ekskuzojn anticipe por kial ne estus profesoro, aŭ la profesoro nur povus fleksi kulerojn.
```

\emph{De kie vi venas, stranga eta antaŭdiro?}
Hari direktis la penson al sia cerbo.
\emph{Kial mi kredas tion, kion mi kredas?}

> Kutime Hari estis sufiĉe bona respondi tiun demandon, sed en ĉi tiu aparta kazo, li havis neniun \emph{ideon} kio lia cerbo pensis.

Author: ```suggestion
Kutime Hari estis sufiĉe bona respondi tiun demandon, sed en ĉi tiu aparta kazo, li havis neniun \emph{ideon}, kion sia cerbo pensis.
```

> Kutime Hari estis sufiĉe bona respondi tiun demandon, sed en ĉi tiu aparta kazo, li havis neniun \emph{ideon}, kion sia cerbo pensis.

Author: ```suggestion
Kutime Hari povis sufiĉe bone respondi tiun demandon, sed en ĉi tiu aparta kazo, li havis \emph{neniun} ideon, kion sia cerbo pensis.
```

> Hari mense levis la ŝultrojn.
> Ebena metala plato sur pordo petas puŝadon, kaj tenilo sur pordo petas tiradon, kaj la afero farenda kun testebla hipotezo estas iri kaj testi ĝin.

Author: ```suggestion
Hari mense levis la ŝultrojn.
Ebena metala plato sur pordo petas puŝadon, kaj tenilo sur pordo petas tiron, kaj la afero farenda kun testebla hipotezo estas iri kaj testi ĝin.
```

Li prenis liniitan paperon de sia skribotablo, kaj komencis skribi.

\begin{writtenNote}
\letterAddress{Estimata Deputita Lernejestrino}
\end{writtenNote}

Hari paŭzis, reflektante; tiam forĵetis la paperon por alia, frapante alian milimetron da grafito el sia mekanika krajono.
Ĉi tio postulis zorgan kaligrafion.

\begin{writtenNote}
\letterAddress{Estimata Deputita Lernejestrino Minerva Mikgonagol,}

\letterAddress{Aŭ Kiu Ajn Povas Esti Koncerninta:}

Mi ĵus ricevis vian akceptan leteron de Hogŭartso, adresita al S\nobreakdash-ro.~H.~Potr.
Vi eble ne scias ke miaj genetikaj gepatroj, Ĝajmz Potr kaj Lili Potr (antaŭe Lili Evanz) estas mortaj.

> Mi estis adoptita per la filino de Lili, Petunja Evanz-Veres, kaj ŝia edzo, Mikael Veres-Evanz.

Author: ```suggestion
Mi estis adoptita per la fratino de Lili, Petunja Evanz-Veres, kaj ŝia edzo, Mikael Veres-Evanz.
```

Mi estas ege interesata por ĉeesti Hogŭartson, kondiĉe de tia loko estas efektive ekzistanta.
Nur mia patrino Petunja diras ke ŝi scias pri magio, kaj ŝi mem ne povas uzi ĝin.
Mia patro estas tre skeptika.
Mi mem estas malcerta.
Mi ankaŭ ne scias kie akiri iun ajn de la libroj aŭ ekipaĵoj listigitaj en via akcepta letero.

Patrino menciis ke vi sendis Hogŭartsan reprezentanton al Lili Potr (tiam Lili Evanz) por demonstri al ŝia familio ke magio estas reala, kaj, mi supozas, por helpi Lili akiri siajn lernejajn materialojn.
Se vi povus fari tion por mia familio, estus tre helpema.

\letterClosing[Sincere,]{Hari Ĝajmz Potr-Evanz-Veres.}
\end{writtenNote}

> Hari aldonis sian nunan adreson, poste faldis la letero Kaj metis ĝin en koverton, kiun li adresis al Hogŭartso.

Author: ```suggestion
Hari aldonis sian nunan adreson, poste faldis la letero kaj metis ĝin en koverton, kiun li adresis al Hogŭartso.
```

> Plia konsidero kondukis lin akiri kandelon kaj gutan vakson sur la klapon de la koverto, en kiu, uzante la pinton de Penknife, li impresis la inicialojn H.Ĝ.P.E.V\@.

Author: ```suggestion
Plia konsidero kondukis lin akiri kandelon kaj gutan vakson sur la klapon de la koverto, en kiu, uzante la pinton de poŝa tranĉilo, li impresis la inicialojn H.Ĝ.P.E.V\@.
```

> % Hari added their current address, then folded up la letero kaj put it in an envelope, which he addressed to Hogŭartso.
> % Further consideration led him to obtain a candle kaj drip wax onto la flap of la envelope, into which, using a penknife’s tip, he impressed la initials H.Ĝ.P.E.V\@.
> % If he was going to descend into this madness, he was going to do it with style.

Author: ```suggestion
Hari aldonis sian nunan adreson, poste faldis la letero Kaj metis ĝin en koverton, kiun li adresis al Hogŭartso.
Plia konsidero kondukis lin akiri kandelon kaj gutan vakson sur la klapon de la koverto, en kiu, uzante la pinton de Penknife, li impresis la inicialojn H.Ĝ.P.E.V\@.
Se li malsupreniros en ĉi tiun frenezon, li farus ĝin kun stilo.
```

Poste li malfermis sian pordon kaj reiris malsupren.

> Lia patro estis sidanta en la salono kaj leganta libron de pli altaj matematikoj por pruvi kiel inteligenta li estis; kaj lia patrino estis en la kuirejo preparanta unu el la plej ŝatata manĝoj de lia patro por pruvi kiel amanta ŝi estis.

Author: ```suggestion
Lia patro estis sidanta en la salono kaj leganta libron de pli altaj matematikoj por pruvi kiel inteligenta li estis; kaj lia patrino estis en la kuirejo preparanta unu el la plej ŝatataj manĝoj de lia patro por pruvi kiel amanta ŝi estis.
```

> % Then he opened his door kaj went back downstairs.
> % His patro was sitting in la living-room kaj reading a book of higher maths to show how smart he was; kaj his patrino was in la kuirejo preparing one of his patro’s favourite meals to show how loving she was.
> % It didn’t look like they were talking to one another at all.
> % As scary as arguments could be, \emph{not arguing} was somehow much worse.

Author: ```suggestion
Poste li malfermis sian pordon kaj reiris malsupren.
Lia patro estis sidanta en la salono kaj leganta libron de pli altaj matematikoj por pruvi kiel inteligenta li estis; kaj lia patrino estis en la kuirejo preparanta unu el la plej ŝatata manĝoj de lia patro por pruvi kiel amanta ŝi estis.
Aspektis, ke ili tute ne parolis unu kun la alia.
Tiel timiga kiel argumentoj povus esti, \emph{ne argumenti} estis iel multe pli malbona.
```

> % “Panjo,” Hari said into la unnerving silence, “I’m going to test la hypothesis.
> % According to your theory, how do I send an owl to Hogŭartso?”

Author: ```suggestion
“Panjo,” Hari diris en la senmovan silenton, “Mi testos la hipotezon.
Laŭ via teorio, kiel mi povas sendi strigon al Hogŭartso?”
```

> % His patrino turned from la kitchen sink to stare at him, looking shocked.
> % “I—I don’t know, I think you just have to own a magic owl.”

Author: ```suggestion
Lia patrino turnis sin de la kuireja lavabo por rigardi lin, aspektante ŝokita.
“Mi—mi ne scias, mi pensas ke vi simple devas posedi magian strigon.”
```

Author: Ĉu eble povas uzi "lavabo" anstataŭ "lavujo"?

Author: Laŭ https://vortaro.net/#lavabo kaj https://vortaro.net/#lavujo, "lavabo" estas "Pelvo, fiksita ĉe la muro de ĉambro aŭ tualetejo, kun krano(j) k defluilo, ofte kompletigita de murspegulo k breto por tualetaĵoj." kaj "lavujo" estas "Tino, pelvo, en kiu oni lavadas tolaĵojn.". Mi pensas ke "lavabo" estas la plej korekta vorto ĉi tie. Jes "lavabo" estas pli simila al "bathroom sink" angle ol "kitchen sink", sed per uzado de "kuireja" mi pensas ke tio estas klara.

> Tio devus soni tre suspektinda, \emph{ho, do ne ekzistas maniero testi vian teorion}, sed la kurioza certeco en Hari ŝajnis volanta elmeti ĝian kolon eĉ plu.

Author: ```suggestion
Tio devus soni tre suspektinde, \emph{ho, do ne ekzistas maniero testi vian teorion}, sed la kurioza certeco en Hari ŝajnis volanta elmeti ĝian kolon eĉ plu.
```

> % That should’ve sounded highly suspicious, \emph{oh, so there’s no way to test your theory then}, but la peculiar certainty in Hari seemed willing to stick its neck out even further.

Author: ```suggestion
Tio devus soni tre suspektinda, \emph{ho, do ne ekzistas maniero testi vian teorion}, sed la kurioza certeco en Hari ŝajnis volanta elmeti ĝian kolon eĉ plu.
```

“Nu, la letero venis ĉi tien iel,” Hari diris, “do mi simple svingos ĝin ĉirkaŭe ekstere kaj krii ‘letero por Hogŭartso!’ kaj rigardu se strigo tenis ĝin.
Paĉjo, ĉu vi volas veni kaj rigardi?”

> % His patro shook his head minutely kaj kept on reading.
> % \emph{Of course,} Hari thought to himself.
> % Magic was a disgraceful thing that only stupid people believed in; if his patro went so far as to \emph{test} la hypothesis, or even \emph{watch} it being tested, that would feel like \emph{associating} himself with that…

Author: ```suggestion
Lia patro skuis la kapon iomete kaj daŭre legis.
\emph{Nature,} Hari pensis al si mem.
% Magic was a disgraceful thing that only stupid people believed in; if his patro went so far as to \emph{test} la hypothesis, or even \emph{watch} it being tested, that would feel like \emph{associating} himself with that…
```

> % Only as Hari stumped out la back door, into la back garden, did it occur to him that if an owl \emph{did} come down kaj snatch la letero, he was going to have some trouble telling Paĉjo about it.

Author: ```suggestion
Nur kiam Hari eliris el la malantaŭa pordo, en la malantaŭan ĝardenon, la penso venis al li, ke se strigo \emph{vere} malsuprenirus kaj prenus la leteron, li havus iom da problemo rakontante al paĉjo pri ĝi.
```

\emph{Sed—nu—tio ne povas \emph{vere} okazi, ĉu ne?
Ne gravas, kion mia cerbo ŝajnas kredi.

> Se strigo vere venas suben kaj tenus ĉi tiun koverton, mi havus zorgojn multe pli gravajn ol kio Paĉjo pensas.}

Author: ```suggestion
Se strigo vere venas suben kaj prenus ĉi tiun koverton, mi havus zorgojn multe pli gravajn ol kio Paĉjo pensas.}
```

Hari profunde enspiris, kaj levis la koverton supren en la aero.

Li glutis.

Kriante \emph{Letero por Hogŭartso!} dum tenanta koverto alte en la aero en la mezo de sia propra ĝardeno estis…efektive tre embarasa, nun, kiam li pensis pri ĝi.

\emph{Ne.
Mi pli bonas ol Paĉjo.
Mi ja uzos la sciencan metodon eĉ se mi sentas min stulta pro tio.}

“Letero—” Hari diris, sed ĝi fakte eliris pli kiel flustra grakado.

> Hari ŝtaligi sian volon, kaj kriis al la malplena ĉielo, \emph{“Letero por Hogŭartso!

Author: ```suggestion
Hari ŝtaligis sian volon, kaj kriis al la malplena ĉielo, \emph{“Letero por Hogŭartso!
```

> Ĉu mi povas havi strigon?”}

Author: ```
Rule: kat_gramatiko
Explanation: La frazo komenciĝas per «Ĉu», sed ne havas demandosignon ĉe la fino de la frazo.

Context: ...por Hogŭartso! Ĉu mi povas havi strigon?”  “Hari, Ĉu tio estas vi?” demandis surp...
```

https://github.com/norwd/hpmor/actions/runs/13190221786/job/36821568759

> Hari ŝtaligis sian volon, kaj kriis al la malplena ĉielo, \emph{“Letero por Hogŭartso!
> Ĉu mi povas havi strigon?”}

Author: ```suggestion
Hari ŝtaligis sian volon, kaj kriis \emph{“Letero por Hogŭartso!
Ĉu mi povas havi strigon?”} al la malplena ĉielo.
```

Author: Ĉi tio trompos (esperante) la `gramma`-an ilon. 🤞

> Hari ŝtaligis sian volon, kaj kriis \emph{“Letero por Hogŭartso!
> Ĉu mi povas havi strigon?”} al la malplena ĉielo.

Author: ```suggestion
Hari ŝtaligis sian volon, kaj kriis al la malplena ĉielo \emph{“Letero por Hogŭartso!
Ĉu mi povas havi strigon?”\phantom{}}
```

> Ĉu mi povas havi strigon?”\phantom{}}

Author: ```suggestion
Ĉu mi povas havi strigon?”}
```

> “Hari, Ĉu tio estas vi?” demandis surprizita voĉo de virino, unu el la najbaroj.

Author: ```suggestion
“Hari, ĉu tio estas vi?” demandis surprizita voĉo de virino, unu el la najbaroj.
```

> Hari tiris malsupren sian manon kiel ĝi brulis kaj kaŝis la koverton malantaŭ sia dorso kiel ĝi estis drogmono.

Author: ```suggestion
Hari tiris malsupren sian manon kvazaŭ ĝi brulis kaj kaŝis la koverton malantaŭ sia dorso kvazaŭ ĝi estis drogmono.
```

Lia tuta vizaĵo estis varma kun honto.

> Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj striitaj per grizo eskapanta el ŝia hareto.

Author: ```suggestion
Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj striitaj per grizo eskapanta el ŝia harreto.
```

Vidu ankaŭ: https://vortaro.net/#harreto

> Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj striitaj per grizo eskapanta el ŝia harreto.

Author: ```suggestion
Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj striitaj per grizo eskapantaj el ŝia harreto.
```

> Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj striitaj per grizo eskapantaj el ŝia harreto.

Author: ```suggestion
Vizaĵo de maljuna virino ekrigardis super la najbara barilo, haroj, striitaj per grizo, eskapantaj el ŝia harreto.
```

S\nobreakdash-ino.~Figg, la fojfoja vartistino.
“Kion vi faras, Hari?”

“Nenio,” Hari diris per sufokita voĉo.
“Nur—testadon de tre stulta teorio—”

“Ĉu vi ricevis vian akceptan leteron de Hogŭartso?”

Hari frostiĝis surloke.

“Jes,” la lipoj de Hari diris iom poste.
“Mi ricevis leteron de Hogŭartso.

> Ili diras ke ili volas mian strigon antaŭ la 31a de julio, sed—”

Author: ```
Rule: typos
Explanation: Ebla mistajpaĵo trovita

Context: ...as ke ili volas mian strigon antaŭ la 31a de julio, sed—”  “Sed vi ne havas strig...
Suggested fix: 1) s  2) ia  3) al  4) la  5) ja  6) da  7) aŭ  8) ha  9) aĥ  10) i
```

https://github.com/norwd/hpmor/actions/runs/13190221786/job/36821568759

Author: ```suggestion
Ili diras, ke ili volas mian strigon antaŭ la 31a de julio, sed—”
```

> Ili diras, ke ili volas mian strigon antaŭ la 31a de julio, sed—”

Author: ```suggestion
Ili diras, ke ili volas mian strigon antaŭ la 31-a de julio, sed—”
```

“Sed vi ne \emph{havas} strigon.
Kompatinda kara!

> Mi ne povas imagi \emph{kion} iu certe pensis, sendinte al vi nur la normon leteron.”

Author: ```suggestion
Mi ne povas imagi \emph{kion} iu certe pensis, sendinte al vi nur la norman leteron.”
```

Sulkigita brako etendis super la barilo, kaj malfermis atendan brakon.
Apenaŭ eĉ pensanta en ĉi tiu punkto, Hari transdonis sian koverton.

“Nur lasu ĝin al mi, kara,” diris S\nobreakdash-ino.~Figg, “kaj mi tuj kondukos iun tien ĉi.”

Kaj ŝia vizaĵo malaperis malantaŭ la barilo.

Estis longa silento en la ĝardeno.

> Tiam knaba voĉo diris, trankvile kaj kviete, “Kio.”

Author: ```suggestion
Tiam la voĉo de knabo diris, trankvile kaj kviete, “Kio.”
```
